### ISSUE TITLE
Feature request: Highlight ideas on the voting/action items stage

### ISSUE THREAD

**Issue by samdec11** [stride-nyc/remote_retro#276]
As a facilitator,
I want the ability to highlight ideas during the voting/action items stage
So that I can bring attention to the idea that I want to discuss as a group.

__By Way Of Explanation:__ When we move from the idea generation to the voting phase, the votes replace all of the idea controls, but the facilitator should still be able to highlight ideas to bring them to everyone's attention

__Acceptance Criteria:__

- [ ] Facilitator can highlight ideas while in the voting and action items stages.

**Comment by vanderhoop** [comment]
hey @samdec11! I've been thinking about this as well. I don't know that we'll want to highlight ideas _during_ the voting stage (as it might allow facilitators to influence the voting thereby), but I can see highlighting providing a ton of value in the action item generation stage. My only concern then is one of design, in that we need real-estate to house the vote interface _and_ the spotlight icon. What do you think about replacing the vote counts from the ideas with the action icons from the idea-generation stage. 

So the flow would be: users vote (just the vote button with vote count)-> facilitator moves them into the action-item stage -> ideas self-sort based on votes DESC -> vote counts disappear, replaced by the old icon set? Is there any downside to this?

**Comment by samdec11** [comment]
Agreed with not needing it on the voting stage, but I don't think the
facilitator should have the edit and delete abilities in the action items
stage (except for on action items) and I still think there's some value in
seeing the vote counts because some teams (like mine) only discuss ideas
that received votes, so there'd be no way to tell which those were if we
completely removed the counts. I think we should try to find a design that
includes the vote counts for everyone and the announce icon for the
facilitator in the action items stage.

**Comment by samdec11** [comment]
@vanderhoop Thoughts on that?

**Comment by vanderhoop** [comment]
> I think we should try to find a design that
includes the vote counts for everyone and the announce icon for the
facilitator in the action items stage.

I agree with this. @imaginista @nonlinear given the current interfaces for highlighting an idea (and the element that--in the action-item stage--just displays votes, is there an design that works better than simply having the elements sit inline?
